cmd/cli/plugin/secret/registry_secret_add.go

```diff
@@ -141,6 +137,27 @@ func updateSecret(cmd *cobra.Command, pkgClient tkgpackageclient.TKGPackageClien
 	return nil
 }
 
+func validateSecretCredentials() error {
```

Comment:
Nit pick: Would it be better to name it `fetchAndValidateSecretCredentials`?

Reply:
Done